vcpkg/readme/vcpkg-README.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Microsoft/vcpkg README
+description: README for vcpkg registry
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: shell-selections
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 10/17/2023
+ms.prod: vcpkg
+ROBOTS: NOINDEX
+---
+
+# vcpkg: Overview
+
+vcpkg helps you manage C and C++ libraries on Windows, Linux and MacOS.
+This tool and ecosystem are constantly evolving, and we always appreciate contributions!
```

Comment:
This is an opportunity to link to the new tutorial: https://learn.microsoft.com/en-us/vcpkg/get_started/get-started-adding-to-registry?pivots=shell-cmd

Also a good idea to link to the maintainer guide: https://learn.microsoft.com/en-us/vcpkg/contributing/maintainer-guide